daemon/server/local_scoot.go

```diff
@@ -0,0 +1,56 @@
+package server
+
+// implement the Scoot API Run requests for local processing
+
+import (
+	"github.com/scootdev/scoot/runner"
+	"github.com/scootdev/scoot/runner/execer"
+	"github.com/scootdev/scoot/runner/local"
+	"github.com/scootdev/scoot/snapshot"
+
+	"fmt"
+	"time"
+)
+
+const maxAccumulatedWaitTime time.Duration = 1 * time.Minute //TODO parameterize this
```

Comment:
the runner needs a timeout value with each command.  I'm also projecting that the daemon's Wait() feature will also have a timeout.  Both can use the same timeout.  I've updated the code to parameterize this with the setup.